src/compo/ompsnm_o3_nc2ioda.py

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env python3
+
+#
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Standard Python library imports.
+import os
+import sys
+import argparse
+import glob
+import netCDF4 as nc
+import numpy as np
+from datetime import datetime, timedelta
+from pathlib import Path
+from collections import defaultdict, OrderedDict
+
+#pyIoda libraries.
+#Append pyioda paths so ioda_conv_engines can be loaded
+IODA_CONV_PATH = Path(__file__).parent/"../lib/pyiodaconv"
+if not IODA_CONV_PATH.is_dir():
+    IODA_CONV_PATH = Path(__file__).parent/'..'/'lib-python'
+sys.path.append(str(IODA_CONV_PATH.resolve()))
+
+import ioda_conv_engines as iconv
+from orddicts import DefaultOrderedDict
+
+
+#Global Dictionaries.
+locationKeyList = [
+    ("latitude", "float"),
+    ("longitude", "float"),
+    ("dateTime", "long"),
+]
+
+ 
+# dictionary to map things we're putting into ioda and taking out of instrument native format
+ioda2nc = {}
```

Comment:
same comment again

Reply:
Again, keeping the dictionary approach from before.